### PR TITLE
feat(HACBS-1668): Release service failing to the container after infra-deployment on aws cluster

### PR DESCRIPTION
**Pull request by naftalysh**
Signed-Off-By: Naftaly Shprai <naftalysh@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,7 +26,9 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 
 # Use ubi-micro as minimal base image to package the manager binary
 # See https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58
-FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
+# FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
+# for debugging purposes, I use my made image in the FROM clause with client utils
+FROM quay.io/nshprai/fedora-35:latest
```
Comment (davidmogar) [inline Dockerfile:31]: Is this a PR for testing? If so, delete the PR please.

File: config/manager/network_policy.yaml [hunk 2]
```diff
@@ -2,11 +2,28 @@
...
 kind: NetworkPolicy
 metadata:
-  name: prevent-all-egress
+  name: prevent-all-egress-allow-api-server
   namespace: system
 spec:
   podSelector:
     matchLabels:
-      control-plane: controller-manager
+      control-plane: controller-man
```
Comment (mmalina) [inline config/manager/network_policy.yaml:10]: Is this intentional? It looks like a mistake.

File: config/manager/network_policy.yaml [hunk 3]
```diff
@@ -2,11 +2,28 @@
...
   policyTypes:
     - Egress
+  egress:
+    - to:
+        - ipBlock:
+            cidr: 172.30.0.0/24
```
Comment (mmalina) [inline config/manager/network_policy.yaml:16]: kubeapi ip can be different on different clusters. I'm not sure what the best approach is. But it doesn't seem great to hardcode it here. Or maybe we can have something here, but then patch it using kustomize in the infra repo. And then also customize it using preview.sh when preparing a testing cluster. @davidmogar wdyt?

Comment (davidmogar) [reply config/manager/network_policy.yaml:16]: This was just a simple workaround that I asked @naftalysh to test. What I don't get is why we have a PR in here for this. Feature branches and forks are for that. I'm closing it.

File: config/manager/network_policy.yaml (hunk 2 above)
Comment (naftalysh) [reply config/manager/network_policy.yaml:10]: This is a mistake , I by mistake deleted the label value, it should be controller-manager

File: config/manager/network_policy.yaml (hunk 3 above)
Comment (naftalysh) [reply config/manager/network_policy.yaml:16]: I can change it to a generic solution

**Pull request closed without merging**